source/CoordinateConversion/ProAppCoordConversionModule/Config.daml

```diff
@@ -1,11 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
 <ArcGIS defaultAssembly="CoordinateConversion.dll" defaultNamespace="ProAppCoordConversionModule" xmlns="http://schemas.esri.com/DADF/Registry" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://schemas.esri.com/DADF/Registry file:///C:/Program%20Files/ArcGIS/Pro/bin/ArcGIS.Desktop.Framework.xsd">
   <AddInInfo id="{dfc85d8b-d2c1-405b-bd03-9a26740d842c}" version="4.0.0.0" desktopVersion="2.2">
     <Name>Coordinate Conversion</Name>
     <Description>Quickly convert coordinates between several formats.</Description>
     <Image>Images\AddinDesktop32.png</Image>
     <Author>Esri</Author>
     <Company>Esri</Company>
-    <Date>6/26/2018</Date>
+    <Date>2/26/2020</Date>
```

Comment:
since the DAML is doing this `<updateModule refID="esri_mapping">`  you probably should have core and mapping added as a dependencies like the following:

`  <dependencies>
    <dependency name="ADCore.daml" />
    <dependency name="ADMapping.daml" />
  </dependencies>
`